Apps/WebClient/src/ClientApp/src/components/timeline/errorCard.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
this looks useful outside the timeline. How about moving it to the components folder (parent folder from this).

Reply:
Sure!

Reply:
Done